requirements.txt

```diff
@@ -17,3 +17,4 @@ pynvml == 11.5.0
 triton >= 2.1.0
 outlines == 0.0.34
 tiktoken == 0.6.0 # Required for DBRX tokenizer
+vllm-nccl-cu12>=2.18<2.19 # for downloading nccl library
```

Comment:
Can we do this?
```suggestion
vllm-nccl-cu12>=2.18.3<2.19 # for downloading nccl library
```

Reply:
Fun fact: actually I have https://pypi.org/project/vllm-nccl-cu12/2.18.1.0.1.0/ , which is nccl 2.18.1 . I don't find nccl 2.18.3 at https://developer.download.nvidia.com/compute/redist/nccl/ .